Tasks/MavenV2/maventask.ts

```diff
@@ -337,7 +338,8 @@ function publishJUnitTestResults(testResultsFiles: string) {
     }
 
     var tp = new tl.TestPublisher("JUnit");
-    tp.publish(matchingJUnitResultFiles, true, "", "", "", true);
+    const testRunTitle = tl.getInput('testRunTitle');
+    tp.publish(matchingJUnitResultFiles, true, "", "", testRunTitle, true, TESTRUN_SYSTEM);
```

Comment:
@madhurig I've added the test run type telemetry to the Maven V2 task so that it matches MavenV3. Is there a reason it was not updated in this PR: https://github.com/Microsoft/azure-pipelines-tasks/pull/7972

Reply:
No particular reason to not add to MavenV2 other than that it was a new feature we added in MavenV3.